src/core/client/framework/lib/network/createManagedSubscriptionClient.ts

```diff
@@ -47,7 +47,7 @@ export interface ManagedSubscriptionClient {
   /** Resume all subscriptions eventually causing websocket to start with new connection parameters */
   resume(): void;
   /** Sets access token and restarts the websocket connection */
```

Comment:
Can you change the comment to: `Sets access token. Note: You have to pause() and resume() the client manually for it to take effect.`